src/sections/Campaign/index.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from "react";
+import has from "lodash/has";
+import PropTypes from "prop-types";
+import { useQuery, useMutation } from "@apollo/react-hooks";
+import Markdown from "markdown-to-jsx";
+import CampaignAction from "../../components/CampaignAction";
+import { GET_USER_ACTIONS, UPDATE_USER_ACTION } from "./api";
+
+const CampaignActions = ({ user, campaignId }) => {
+  const [completedActions, setCompletedActions] = useState({});
+  const [completeAction, { data: mutationResponse }] = useMutation(
+    UPDATE_USER_ACTION
+  );
+  const {
+    loading: queryLoading,
+    error: queryError,
+    data: queryResponse
+  } = useQuery(GET_USER_ACTIONS, {
+    variables: { campaignId, userId: user.id }
+  });
+
+  useEffect(() => {
```

Comment:
This doesn't feel right to me, having to track this ourselves seems like an issue. I think we should leave this to Apollo instead.

Maybe the answer to this is here in their documentation https://www.apollographql.com/docs/react/data/mutations/#updating-the-cache-after-a-mutation

Reply:
You are totally right. I didn't think about it. will do in a next iteration https://github.com/debtcollective/student-debt-campaign-netlify/issues/18